design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
+    ): UpdatedPricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    assignBillingAddressToNegotiableQuote(
+        billingAddress: NegotiableQuoteBillingAddressInput!
+    ): AssignBillingAddressToNegotiableQuoteOutput @doc(description:"Assigns new billing address to negotiable quote")
+
+    estimateShippingCostForNegotiableQuote(
+        shippingAddress: NegotiableQuoteEstimateShippinAddressInput!
+    ): EstimateShippingCostForNegotiableQuoteOutput @doc(description:"Estimate shipping costs for a negotiable quote")
+
+    estimateShippingCostByAddressId(
```

Comment:
Are all these `estimate` fields mutating something in Magento? I'm not _super_ familiar with how B2B UI works, but surprised that `estimate` would be mutative and not just a query

Reply:
All the estimates have been changed to query from mutation